rpcs3/rpcs3qt/gui_application.cpp

```diff
@@ -941,8 +941,13 @@ void gui_application::OnChangeStyleSheetRequest()
 	if (stylesheet_name.isEmpty() || stylesheet_name == gui::DefaultStylesheet)
 	{
 		gui_log.notice("Using default stylesheet");
+#ifdef __APPLE__
```

Comment:
i don't see a reason for this change either.
you're already setting the default to native, which will never reach this place.